application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -84,14 +98,62 @@ public void onSlashCommand(@NotNull final SlashCommandEvent event) {
             logger.warn(
                     "Unable to render latex, could not convert the image into an attachable form. Formula was {}",
                     latex, e);
-            return;
+            return null;
+        }
+        return renderedTextImageStream;
+    }
+
+    @Nullable
+    private Image getImage(@NotNull SlashCommandEvent event, String latex, TeXFormula formula) {
```

Comment:
I dont like the name of this. It is not a getter. It computes some heavy stuff.

It takes the `latex` and gives out an `Image` for it. I would call it `renderImage` instead. Or `renderLatexAsImage`.

Reply:
While at it, please change the order of the methods to the order they appear in the code. So:
1. getImage
2. getRenderedTextImageStream
3. sendImage